js-exercise/assets/js/script.js

```diff
@@ -0,0 +1,87 @@
+console.log(1 + 2);
```

Comment:
Scrpt file naminf convention -> File would be generally named index.js or main.js and placed in a folder "scripts". in the same hierarchy as css folder